frontend/src/stores/useBiolinkModel.js

```diff
@@ -47,57 +53,57 @@ export default function useBiolinkModel() {
   }
 
   /**
-   * Get a list of ancestors for all biolink classes
+   * Get all descendants by getting direct descendants recursively
+   * @param {string} parentClass - biolink class to get direct descendants of
+   * @param {object} classes - object of all biolink classes
+   * @returns {string[]} list of all descendants
+   */
+  function getDirectDescendants(parentClass, classes) {
```

Comment:
```suggestion
  function getDescendants(parentClass, classes) {
```

Reply:
Fixed in a911937f47e64394d40209e43c9462292db59476